packages/next/src/shared/lib/app-router-context.shared-runtime.ts

```diff
@@ -66,6 +66,13 @@ export type LazyCacheNode = {
    * Child parallel routes.
    */
   parallelRoutes: Map<string, ChildSegmentMap>
+
+  /**
+   * The timestamp of the navigation that last updated the CacheNode's data. If
+   * a CacheNode is reused from a previous navigation, this value is not
+   * updated. Used to track the staleness of the data.
+   */
+  navigatedAt: number
```

Comment:
As it seems the case that `-1` is passed in here, I'd advocate instead for using `null` or an explicit `undefined` to avoid using a magic number.

Reply:
Logically it’s equivalent to a navigation that happened infinitely long ago, and is therefore always stale. Like it might as well be -Infinity. There aren’t any explicit comparisons anywhere so I don’t consider it a magic number.